frontend/shared/src/types/employer-application.d.ts

```diff
@@ -0,0 +1,11 @@
+import type Status from 'shared/types//application-status';
+import type Invoicer from 'shared/types//invoicer';
```

Comment:
What are these double slashes?

Reply:
good catch! Interesting that there were no errors